pr-data.csv

```diff
@@ -1394,7 +1393,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndRemoveClosure_failureBranch,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryExists,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-api,org.apache.ignite.internal.storage.ConcurrentHashMapStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
```

Comment:
Please do not just delete the old info. Per https://github.com/TestingResearchIllinois/idoft#how-to-indicate-that-a-test-is-renamed-or-moved-to-a-different-location change the status to `MovedOrDeleted` and then add a new test.

Reply:
Updated as requested. Thanks.